docs/source/tutorial/cli/train.rst

```diff
@@ -61,7 +61,7 @@ Our code supports several methods of splitting data into train, validation, and
 
 * **Separate val/test:** If you have separate data files you would like to use as the validation or test set, you can specify them with :code:`--separate-val-path <val_path>` and/or :code:`--separate-test-path <test_path>`.
 
-*Note*: By default, both random and scaffold split the data into 80% train, 10% validation, and 10% test. This can be changed with :code:`--split-sizes <train_frac> <val_frac> <test_frac>`. The default setting is :code:`--split-sizes 0.8 0.1 0.1`. Both splits also involve a random component that can be seeded with :code:`--seed <seed>`. The default setting is :code:`--seed 0`.
+Note: By default, both random and scaffold split the data into 80% train, 10% validation, and 10% test. This can be changed with :code:`--split_sizes <train_frac> <val_frac> <test_frac>`. For example, the default setting is :code:`--split_sizes 0.8 0.1 0.1`. Both also involve a random component and can be seeded with :code:`--data-seed <seed>`. The default setting is :code:`--data-seed 0`.
```

Comment:
Is it `--split-sizes` or `--split_sizes`? I thought we use `-` in v2?

Reply:
Good catch, I must have missed that when merging v2/dev into my branch.